package.json

```diff
@@ -41,14 +41,14 @@
     "fast-redact": "^3.5.0",
     "pino": "^9.8.0",
     "pino-std-serializers": "^7.0.0",
-    "pure-parse": "^1.0.1"
+    "sury": "11.0.0-alpha.3"
   },
   "devDependencies": {
     "@changesets/cli": "2.29.5",
     "@changesets/get-github-info": "0.6.0",
     "@types/fast-redact": "3.0.4",
     "@types/split2": "4.2.3",
-    "skuba": "12.1.1",
+    "skuba": "12.2.0-main-20250826233130",
```

Comment:
Does this mean that consumers also need to upgrade to `skuba@12.2.0` to avoid Jest module mapper issues? If so, we may want to hold off on this release until folks have had time to adopt that new skuba version?

Reply:
I'm thinking about bundling just `sury` with tsdown?